web/client/epics/playback.js

```diff
@@ -30,7 +32,13 @@ const BUFFER_SIZE = 20;
 const PRELOAD_BEFORE = 10;
 const domainArgs = (getState, paginationOptions = {}) => {
     // const timeData = timeDataSelector(getState()) || {};
-    return ['http://cloudsdi.geo-solutions.it:80/geoserver/gwc/service/wmts', "landsat8:B3", "time", {
+    const selectedLayer = selectedLayerSelector(getState());
+    const layerName = getLayerFromId(getState(), selectedLayer).name;
+
+
+    const layerUrl = getLayerFromId(getState(), selectedLayer).dimensions.filter((x) => x.name === "time").map((l) => l.source.url);
+
+    return [layerUrl, layerName, "time", {
```

Comment:
take also into account that no layer selection is possible. For the moment, if this happens, do nothing.